src/lib/components/ui/Text/Text.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withProviderContext } from '../../../provider';
+import getRootHyphensClassName from './helpers/getRootHyphensClassName';
+import getRootWordWrappingClassName from './helpers/getRootWordWrappingClassName';
+import styles from './Text.scss';
+
+export const Text = ({
+  blockLevel,
+  children,
+  hyphens,
+  id,
+  lines,
+  wordWrapping,
+}) => {
+  const HtmlElement = blockLevel ? 'div' : 'span';
+
+  return (
+    <HtmlElement
+      className={(hyphens !== 'none' || lines > 0 || wordWrapping !== 'normal')
+        ? [
+          (lines > 0) ? styles.rootClampLines : '',
+          getRootHyphensClassName(hyphens, styles),
+          getRootWordWrappingClassName(wordWrapping, styles),
+        ].join(' ')
+        : undefined}
+      id={id}
+      style={(lines > 0) ? { '--rui-custom-lines': lines } : undefined}
+    >
+      {children}
+    </HtmlElement>
+  );
+};
+
+Text.defaultProps = {
+  blockLevel: false,
+  children: null,
+  hyphens: 'none',
+  id: undefined,
+  lines: undefined,
+  wordWrapping: 'normal',
+};
+
+Text.propTypes = {
+  /**
+   * If true, the root HTML element renders as `<div>` instead of `<span>`.
+   */
+  blockLevel: PropTypes.bool,
+  /**
+   * Text content to be sanitized. Can contain HTML.
```

Comment:
I think it should be `Can contains HTML`

Reply:
Only if we were talking about [cans](https://slovnik.seznam.cz/preklad/cesky_anglicky/plechovka?strict=true) 🙂.